tasks/yum/install.yml

```diff
@@ -0,0 +1,39 @@
+- name: "Make sure Jenkins is stopped"
+  service: name="jenkins" state="stopped"
+  ignore_errors: yes
+- name: "Create Jenkins group"
+  group:
+    name: "{{ jenkins_config_group }}"
+    state: present
+- name: "Create Jenkins user"
+  user: name="{{ jenkins_config_owner }}" groups="{{ jenkins_config_group }}" state="present"
+- name: "Install Java"
+  yum: name="java" state="present"
+- name: "Install git"
+  yum: name="git" state="present"
+- name: "Add Jenkins repository"
+  get_url: url="http://pkg.jenkins-ci.org/redhat/jenkins.repo" dest="/etc/yum.repos.d/jenkins.repo"
+- name: "Import Jenkins key"
+  command: "rpm --import https://jenkins-ci.org/redhat/jenkins-ci.org.key"
+- name: "Install Jenkins"
+  yum:
+    name: jenkins
+    state: present
+- name: Set JENKINS_HOME
+  lineinfile:
+    create: "yes"
+    dest: "/etc/default/jenkins"
+    line: "JENKINS_HOME={{ jenkins_home }}"
+    regexp: '^JENKINS_HOME='
+    state: "present"
+- name: "Set Jenkins command line options"
+  lineinfile:
+    dest: /etc/default/jenkins
+    regexp: '^JAVA_ARGS='
+    line: "JAVA_ARGS=\"{{ jenkins_java_opts }}\""
+- name: "Configure the firewall to allow Jenkins port"
+  command: "firewall-cmd --zone=public --add-port={{ jenkins_port }}/tcp --permanent"
```

Comment:
This shouldn't be in this role because we don't want to couple this role with a certain firewall. If you want to achieve this you can use a separate firewall role (lots already exist).

Can you remove these tasks that configure the firewall please?